db/migrate/20220217104747_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :User do |t|
+      t.string :Name
+      t.string :Photo
+      t.text :Bio
```

Comment:
- [ ] <b>REQUIRED:</b> I didn't mention this in the previous review, but It would be nice if you change those too to `lower case` before you merge.

Reply:
> * [ ]  REQUIRED: I didn't mention this in the previous review, but It would be nice if you change those too to `lower case` before you merge.

Thank you so much @rukundoeric Am going to correct it before merging